src/Symfony/Component/String/TruncateCut.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\String;
+
+enum TruncateCut
```

Comment:
What about char, beforeWord, afterWord instead?
I don't like the "strict" word: it looks like a political or religious promise (think "strict types")

Reply:
What do you think about the new enum cases I used ?